src/whiteboard-utility.js

```diff
@@ -162,104 +121,124 @@ class WhiteboardUtility {
   }
 
   activeElement(ev, tool) {
-    const prevSelectedTool = document.querySelector(`#t_${tool.type}${virtualclass.gObj.currWb} .selected`);
+    const wbId = virtualclass.gObj.currWb;
+    const prevSelectedTool = document.querySelector(`#t_${tool.type}${wbId} .selected`);
     if (prevSelectedTool != null) {
       prevSelectedTool.classList.remove('selected');
     }
 
     const currElementValue = ev.target.dataset[tool.prop];
     if (currElementValue != null) {
       ev.target.classList.add('selected');
-      this.changeToolProperty(tool.type, currElementValue, virtualclass.gObj.currWb);
+      this.constructor.updateToolStyle(tool.type, currElementValue, wbId);
       if (tool.type === 'color') {
-        document.querySelector(`#t_color${virtualclass.gObj.currWb} .disActiveColor`).style.backgroundColor = virtualclass.wb[virtualclass.gObj.currWb].activeToolColor;
+        document.querySelector(`#t_color${wbId} .disActiveColor`).style.backgroundColor = virtualclass.wb[wbId].toolColor;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 122. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5384064)